AutomatedTesting/Gem/PythonTests/AWS/README.md

```diff
@@ -46,8 +52,16 @@
             python/python.sh -m pytest {path_to_the_test_file} --build-directory {directory_to_the_profile_build}
         ```
 
-### Pycharm
-You can also run any specific automation test directly from Pycharm by providing the "--build-directory" argument in the Run Configuration.
+The main tests files can be found under this directory at the following paths:

```

Comment:
thank you for adding this! very handy

Reply:
It slowed me down trying to understand where these files so though it had value.